cmd/cmd.go

```diff
@@ -92,13 +94,86 @@ var newAddress = &cobra.Command{
 	},
 }
 
-
 var burn = &cobra.Command{
-	Use:"burn",
-	Short:"Burns the specied amount of FCT into PNT",
-	Args: CombineCobraArgs(CustomArgOrderValidationBuilder(true, ArgValidatorFCTAddress, ArgValidatorFCTAmount)),
+	Use:   "burn <fct address> <fct amount>",
+	Short: "Burns the specied amount of FCT into PNT",
+	Long: "Burning FCT will turn it into PNT. The PNT burn address is an EC address, and the transaction has " +
+		"an input with # of FCT, and an output of 0 EC. This means the entire tx input becomes the fee. " +
+		"This command costs FCT, so be careful when using it.",
+	Example: "pegnet burn FA3EPZYqodgyEGXNMbiZKE5TS2x2J9wF8J9MvPZb52iGR78xMgCb 1",
+	// TODO: Verify this functionality.
+	ValidArgs: ValidOwnedFCTAddresses(),
+	Args:      CombineCobraArgs(CustomArgOrderValidationBuilder(true, ArgValidatorFCTAddress, ArgValidatorFCTAmount)),
 	Run: func(cmd *cobra.Command, args []string) {
+		defer factom.DeleteTransaction("burn") // Any cleanup from errors
```

Comment:
"burn" should not be harcoded separately from the variable line 131. Opens up a high risk that one reference is updated without the other

Reply:
Done :+1: